packages/time-picker/src/common/picker.vue

```diff
@@ -177,6 +178,12 @@ const valueEquals = function(a, b) {
   return false
 }
 
+const PLACEMENT_MAP = {
+  left: 'bottom-start',
+  center: 'bottom',
+  right: 'bottom-end',
+}
+
 export default defineComponent({
```

Comment:
Seems this config is not valid in the new popperjs@core?

Reply:
![image](https://user-images.githubusercontent.com/15975785/110789904-5ad72600-82ab-11eb-9e01-001f1853585b.png)
[Popperjs/@core](https://popper.js.org/docs/v2/constructors/)
Seems legit.

Reply:
pending？？